### DISCUSSION TITLE
Can we adjust the perspective of the isometric plane and everything inside?

### DISCUSSION THREAD

**Discussion by TeaRexcellence** [elchininet/isometric#145] [Q&A]
I just started playing with it. Its really fun.  Is the isometric grid/plane defaulted to a 30-degree perspective? And can i change that anywhere? That would be game changing.

**Comment by elchininet** [comment]
Hi @TeaRexcellence,
No, it is not possible to build graphics with perspective, the library is to create isometric projections with 30 degress angle from the vertical axes.
Regards

>That would be game changing.

Yes, that would be cool 😉

**Comment by TeaRexcellence** [comment]
Okay so while i wasnt able to achieve free view perspective changes i did achieve the perspective i needed for the build im doing.
https://github.com/TeaRexcellence/isometric-26.565

Cheers man! Love this library

**Reply by elchininet** [reply]
That‘s great! 😃
